Bottom: b48b7f3f1d4422954590ec6bae54096c69460cb4
Top:    86e94e02299f806406fd359322af4f6b63c74671
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 16:11:12 +0100

Refresh of support-holes-in-input-polygon

---

diff --git a/src/polygon.h b/src/polygon.h
index 38777b1..6b23e4d 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,6 +34,7 @@
 #include "global.h"
 
 Cardinal polygon_point_idx (PolygonTypePtr polygon, PointTypePtr point);
+Cardinal polygon_point_contour (PolygonTypePtr polygon, Cardinal point);
 Cardinal prev_contour_point (PolygonTypePtr polygon, Cardinal point);
 Cardinal next_contour_point (PolygonTypePtr polygon, Cardinal point);
 Cardinal GetLowestDistancePolygonPoint (PolygonTypePtr,
